src/configobj/__init__.py

```diff
@@ -1951,7 +1951,7 @@ def _set_configspec(self, section, copy):
                 section[entry].configspec = configspec[entry]
 
 
-    def _write_line(self, indent_string, entry, this_entry, comment):
+    def _write_line(self, indent_string, entry, this_entry, comment, assignment_delim=' = '):
```

Comment:
Repeating the default in the private method should be avoided, i.e. remove the `=…` please.